test/tests/functional/pbs_apbasil_inventory.py

```diff
@@ -43,34 +43,39 @@
 
 
 @tags('cray', 'mom')
-class TestBasilInventory(TestFunctional):
+class TestBasilQuery(TestFunctional):
     """
-    This test suite is for testing the support for BASIL 1.4 inventory
+    This test suite is for testing the support for BASIL 1.7/1.4 basil
     query.Test if query is made with correct BASIL version, and that
     vnodes are getting created as per the query response.
     """
-    basil_version = ['1.4', '1.3', '1.2', '1.1']
+    basil_version = ['1.7', '1.4', '1.3']
```

Comment:
Why did you remove '1.2' and '1.1'?

Reply:
Cause there was nothing to test for these version.